var/spack/repos/spack_repo/builtin/packages/py_h5py/package.py

```diff
@@ -64,13 +68,15 @@ class PyH5py(PythonPackage):
     depends_on("py-setuptools", type="build")
 
     # Build and runtime dependencies
-    depends_on("py-numpy@1.19.3:", type=("build", "run"), when="@3:3.5,3.12: ^python@3.9.0:")
+    depends_on("py-numpy@2:3", type=("build", "run"), when="@3.11: ^python@3.9:")
```

Comment:
Do you mean to include `py-numpy@3` in this range? Version ranges in spack are inclusive. Maybe you meant:
```suggestion
    depends_on("py-numpy@2", type=("build", "run"), when="@3.11: ^python@3.9:")
```
which would then be only `py-numpy` version 2, excluding future version 3.

Reply:
yes whoops, meant to be `2:2` but I think `@2` is sufficient then